ts.js

```diff
@@ -137,63 +130,7 @@
             "allowConciseArrowFunctionExpressionsStartingWithVoid": false
          }
       ],
-      '@typescript-eslint/no-inferrable-types': ['error', {'ignoreParameters': true}],
-      'arrow-body-style': [
-         'error', 
-         'as-needed'
-      ],
-      'complexity': 'off',
-      'constructor-super': 'error',
-      'curly': 'error',
-      'guard-for-in': 'error',
-      'id-blacklist': [
-         'error',
-         'any',
-         'Number',
-         'number',
-         'String',
-         'string',
-         'Boolean',
-         'boolean',
-         'Undefined',
-         'undefined'
-      ],
-      'id-match': 'error',
-      'max-classes-per-file': [
-         'error',
-         1
-      ],
-      'max-len': 'off',
-      'no-caller': 'error',
-      'no-cond-assign': 'error',
-      'no-debugger': 'error',
-      'no-empty': 'error',
-      'no-fallthrough': 'off',
-      'no-invalid-this': 'off',
-      'no-underscore-dangle': 'error',
-      'no-unsafe-finally': 'error',
-      'no-unused-labels': 'error',
-      'prefer-arrow/prefer-arrow-functions': 'error',
-      'space-before-function-paren': [
-         'error',
-         {
-            'anonymous': 'never',
-            'asyncArrow': 'always',
-            'named': 'never'
-         }
-      ],
-      'spaced-comment': [
-         'error',
-         'always',
-         {
-            'markers': [
-               '/'
-            ]
-         }
-      ],
-      'use-isnan': 'error',
-      'valid-typeof': 'off',
-      'linebreak-style': 'off'
```

Comment:
same question as above, these 3 were default and therefore no longer show in the file at all?

Reply:
I don't see a good reason why `linebreak-style` should be off in TS so I just changed that. As for the other two, they both come from `eslint:recommended` so they're redundant